solr/solrj/src/java/org/apache/solr/common/util/ContentStream.java

```diff
@@ -19,10 +19,13 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.Reader;
+import org.apache.solr.client.solrj.request.RequestWriter;
 
 /**
  * @since solr 1.2
+ * @deprecated Use {@link RequestWriter.ContentWriter} instead.
```

Comment:
This causes the CI test to fail due to `Javadoc: Invalid member type qualification`. I think `@deprecated Use {@link ContentWriter} instead`. should work fine